tests/graphql_spec.rs

```diff
@@ -151,7 +168,15 @@ impl GraphQLSpec {
       if path.is_file() && path.extension().unwrap_or_default() == "graphql" {
         let contents = fs::read_to_string(path.clone())?;
         let path_buf = path.clone();
-        files.push(GraphQLSpec::new(path_buf, contents.as_str()));
+        let spec = GraphQLSpec::new(path_buf, contents.as_str());
+
+        match spec.annotation {
+          Some(Annotation::Only) | None => files.push(spec),
+          Some(Annotation::Skip) => {
+            log::warn!("Skipping spec: {}", spec.path.display());
+          }
+          Some(Annotation::Fail) => {}
```

Comment:
The annotation `Fail` should be designed so that it passes when the assertion fails, and conversely, it should fail if the assertion succeeds.